### PR TITLE
admin users tutorials

### PR DESCRIPTION
**Pull request by mishaschwartz**
Tutorials for managing users and permissions for admins.

I decided to put them all on one page since that made more sense in my head. Feel free to suggest a different arrangement.

### CODE REVIEW

**Review by alexandercyu — APPROVED** [review]
Good write-up and well organized.

File: tutorials/admin/users/users.md [hunk 1]
```diff
@@ -1 +1,399 @@
...
+
+(service-permissions-api)=
+#### api
+
+This service type allows you to define access rules based on URL path segments.
+
+For example, the `STAC` service has the following endpoints:
+
+- stac/
+- stac/collections
```
Comment (dchandan) [inline tutorials/admin/users/users.md:313]: Does this pattern also include something like "stac/collections/{collection_id}/items"?

File: tutorials/admin/users/users.md [hunk 2]
```diff
@@ -1 +1,399 @@
...
+Permissions for api services can either be:
+
+- `read`: allow/deny users to make GET and HEAD requests to the endpoint
+- `write`: allow/deny users to make all other requests methods to the endpoint
+
+#### geoserver
+
+This service type is used to apply permissions for the [Geoserver](https://geoserver.org/) service.
+
+This service type will only be used if the [included geoserver component is enabled](../services/services.md).
```
Comment (dchandan) [inline tutorials/admin/users/users.md:327]: Maybe I recall incorrectly, but I don't think the version of services tutorial you have mentioned geoserver or any other services by name.

File: tutorials/admin/users/users.md [hunk 3]
```diff
@@ -1 +1,399 @@
...
+
+For a list of geoserver operations and their description please refer to the geoserver documentation for:
+
+- [Web Feature Service](https://docs.geoserver.org/latest/en/user/services/wfs/reference.html#operations)
+- [Web Mapping Service](https://docs.geoserver.org/latest/en/user/services/wms/reference.html#operations)
+- [Web Processing Service](https://docs.geoserver.org/latest/en/user/services/wps/operations.html)
+
+#### thredds
+
+This service type is used to apply permissions for the [Thredds](https://www.unidata.ucar.edu/software/tds/) service.
```
Comment (dchandan) [inline tutorials/admin/users/users.md:344]: Thredds should be "THREDDS"

File: tutorials/admin/users/users.md [hunk 4]
```diff
@@ -1 +1,399 @@
...
+And you can [define an access rule](permissions-child-resource) for each!
+
+Permissions for api services can either be:
+
+- `read`: allow/deny users to make GET and HEAD requests to the endpoint
+- `write`: allow/deny users to make all other requests methods to the endpoint
+
+#### geoserver
+
+This service type is used to apply permissions for the [Geoserver](https://geoserver.org/) service.
```
Comment (dchandan) [inline tutorials/admin/users/users.md:325]: Should be "GeoServer", same for other instances in this section.

File: tutorials/admin/users/users.md [hunk 5]
```diff
@@ -1 +1,399 @@
...
+
+Here you will see a form to add a new user to your node:
+
+```{image} images/magpie-add-user-form.png
+:alt: Magpie Add Users Form
+:width: 300px
+:align: center
+```
+
+The "User name" and "Email" values must be unique and cannot include some special characters. The "Password" value must
```
[the remark this answers is not in the document]
Comment (mishaschwartz) [reply tutorials/admin/users/users.md:63]: yes

File: tutorials/admin/users/users.md [hunk 6]
```diff
@@ -1 +1,399 @@
...
+variable to `True` in the `env.local` file.
+
+If you do this, we recommend [monitoring](../monitoring/monitoring.md) your node closely to ensure that your node's 
+resources are not over-taxed if too many users are logged in at once.
+```
+
+(update-user-data)=
+### Updating User Data
+
+In general, we don't recommend changing a user's username, password, or email address yourself. Instead, encourage your
```
[the remark this answers is not in the document]
Comment (mishaschwartz) [reply tutorials/admin/users/users.md:90]: Good point, a user can't change their own username, only an admin can do that. I'll update this with that in mind.

File: tutorials/admin/users/users.md [hunk 7]
```diff
@@ -1 +1,399 @@
...
+:alt: Magpie Edit Users Button
+:width: 100px
+:align: center
+```
+
+Then select the user you would like to edit from the table and click the "Edit" button (in the "Action" column).
+Select one of the "Edit" buttons beside the Username, Password, or Email fields and change the value as desired.
+When finished, click the "Save" button to save the changes. 
+
+```{warning}
```
[the remark this answers is not in the document]
Comment (mishaschwartz) [reply tutorials/admin/users/users.md:106]: They all can be used as login credentials. A user can log in with username + password OR email + password

File: tutorials/admin/users/users.md [hunk 8]
```diff
@@ -1 +1,399 @@
...
+The "Group name" value must be unique. If you choose to make the group discoverable, it's name and description will be
+visible to all users. Otherwise, it will only be visible to administrator users.
+
+```{note}
+An administrator user can add any user to any group, whether the group is "discoverable" or not.
+```
+
+If you would like the user to have to agree to some terms and conditions before joining the group, you can add those to
+the "Terms and conditions" field. This is useful, for example, when members of this group are given access to a resource
+that requires a special licensing agreement.
```
[the remark this answers is not in the document]
Comment (mishaschwartz) [reply tutorials/admin/users/users.md:173]: Yeah I meant to tell you about it!

File: tutorials/admin/users/users.md (hunk 1 above)
Comment (mishaschwartz) [reply tutorials/admin/users/users.md:313]: Yes, I just used these as an example. I can throw an "etc." on the end to show that the list isn't exhaustive.

File: tutorials/admin/users/users.md (hunk 2 above)
Comment (mishaschwartz) [reply tutorials/admin/users/users.md:327]: No, not yet at least until I can get the birdhouse documentation up to date.
Do you think I should remove the link until then?

File: tutorials/admin/users/users.md (hunk 7 above)
[the remark this answers is not in the document]
Comment (dchandan) [reply tutorials/admin/users/users.md:106]: Okay I see what you mean.

File: tutorials/admin/users/users.md (hunk 2 above)
Comment (dchandan) [reply tutorials/admin/users/users.md]: I leave that to you. I am totally good with leaving it here if that helps you keep track of this.

**Review by dchandan — APPROVED** [review]

**Pull request merged**